www/app/DashboardController.js

```diff
@@ -3967,16 +3967,16 @@ define(['app'], function (app) {
 			$('#dashcontent .dimslider').slider({
 				//Config
 				range: "min",
-				min: 1,
-				max: 16,
-				value: 5,
+				min: 0,
+				max: 15,
```

Comment:
this is of course cosmetic as it would be set below in the create function ;)

Reply:
Not purely cosmetic because `min` is not set by create function.

Reply:
true ;)